src/main/java/attendance/dto/AttendanceDTO.java

```diff
@@ -0,0 +1,40 @@
+package attendance.dto;
+
+import attendance.model.AttendanceDetail;
+import attendance.model.AttendanceWarning;
+import attendance.model.Crew;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public record AttendanceDTO(
```

Comment:
보통 Dto 네이밍은 아래와 같이 합니다 😄 

```suggestion
public record AttendanceDto(
```

HttpUrlCsvDto 와 같은 축약 형태로만 이루어진 네이밍의 경우 어떻게 될까요?

Reply:
모두 대문자로 표기하는 경우도 있어서 크게 신경쓰지 않았는데 3글자 이상의 축약어의 경우에는 앞글자만 대문자로 표기한다는 컨벤션이 일반적임을 알았습니다. VO같은 경우 2글자라서 모두 대문자로 써도 된다는 것도 알게 되었어요..!

컨벤션은 일관성이 중요한데, HttpUrlCsvDto는 모두 축약어이기 때문에 DTO를 모두 대문자로 표기한다고 정하게 된다면 HTTPURLCSVDTO라는 가독성 없는 클래스명이 탄생할 수도 있겠네요..😅